Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -66,7 +66,7 @@ class FixCommand extends Command
     /**
      * Config instance.
```

Comment:
it's not a type of content but content itself described here.
Therefore sth like `instance of class implementing ConfigInterface`... so long ;)